src/clr_bundle_ls.c

```diff
@@ -54,14 +56,14 @@ static void print_help(void)
 
 	print("Options:\n");
 	print("   -a, --all               List all available bundles for the current version of Clear Linux\n");
-	print("   -d, --deps=[BUNDLE]     List bundles included by BUNDLE\n");
 	print("   -D, --has-dep=[BUNDLE]  List dependency tree of all bundles which have BUNDLE as a dependency\n");
+	print("   --deps=[BUNDLE]         List bundles included by BUNDLE\n");
 	print("\n");
 }
 
 static const struct option prog_opts[] = {
 	{ "all", no_argument, 0, 'a' },
-	{ "deps", required_argument, 0, 'd' },
+	{ "deps", required_argument, 0, DEPS_FLAG },
```

Comment:
maybe we should port (later, not in this PR) all cases that we have longopts only to use this strategy. it's way better for the config loader. What do you think?
I though it's way cleaner

Reply:
yes, we can do that